nvflare/private/fed/app/client/worker_process.py

```diff
@@ -129,17 +134,17 @@ def main():
         client_app_runner.start_run(app_root, args, config_folder, federated_client, secure_train)
 
     except BaseException as e:
-        logger = logging.getLogger("worker_process")
-        logger.error(f"FL client execution exception: {secure_format_exception(e)}")
+        if logger:
+            logger.error(f"FL client execution exception: {secure_format_exception(e)}")
```

Comment:
it is possible the error happens during 
```
conf = FLClientStarterConfiger(
            workspace=workspace,
            args=args,
            kv_list=args.set,
        )
        conf.configure()

```

Reply:
When this happened, the logger has not been initialized, we could not use the logger to log the error. Instead, the exception will be raised. the Pyhton will print out the clear program stack trace.

Reply:
the old logic will log error out securely in that case, we might want to keep the old logic